javascript/javascript2/week3/home-work.js

```diff
@@ -0,0 +1,116 @@
+// Log out the text Called after 2.5 seconds 2.5 seconds after the script is loaded.
+// Create a function that takes 2 parameters: delay and stringToLog.Calling this function
+//     should log out the stringToLog after delay seconds.C
+// all the function you have created with some different arguments.
+// Create a button in html.When clicking this button, use the function you
+// created in the previous task to log out the text: Called after
+//  5 seconds 5 seconds after the button is clicked.
+function delayMsg(delay, stringToLog) {
+    setTimeout(() => {
+        console.log(`Called after 5 seconds`);
+    }, delay * 1000)
+}
+const printMSg = delayMsg;
```

Comment:
This step could be avoided if the objective is to keep a shorter code. But it is great that you know that a function can be assigned to a variable, so feel free to leave it this way :)